src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -148,32 +149,43 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
 
                             using (var fs = new FileStream(output.FullName, FileMode.Create, FileAccess.Write))
                             {
-                                Console.Out.WriteLine($"Process        : {process.MainModule.FileName}");
-                                Console.Out.WriteLine($"Output File    : {fs.Name}");
+                                systemConsole.Out.WriteLine($"Process        : {process.MainModule.FileName}");
+                                systemConsole.Out.WriteLine($"Output File    : {fs.Name}");
                                 if (shouldStopAfterDuration)
-                                    Console.Out.WriteLine($"Trace Duration : {duration.ToString(@"dd\:hh\:mm\:ss")}");
+                                    systemConsole.Out.WriteLine($"Trace Duration : {duration.ToString(@"dd\:hh\:mm\:ss")}");
+
+                                systemConsole.Out.WriteLine("\n");
+
+                                var region = new Region(0, systemConsole.CursorTop, 55, 1, true);
+                                var renderer = new ConsoleRenderer(systemConsole, resetAfterRender: true);
+                                systemConsole.Out.WriteLine("\nPress <Enter> or <Ctrl+C> to exit...");
 
-                                Console.Out.WriteLine("\n\n");
                                 var buffer = new byte[16 * 1024];
 
+                                Task.Run(() =>
+                                {
+                                    while (true)
```

Comment:
This looks like a busy loop. Lets use a timed delay to update at a measured rate